sdk/builder.go

```diff
@@ -245,12 +245,14 @@ func (b *Builder) buildPreReleaseString() (string, error) {
 	}
 
 	// check if cluster app has a pre-release
-	clusterAppPreReleaseString, err := versionStringHasPreReleaseString(b.clusterApp.Version)
-	if err != nil {
-		return "", microerror.Mask(err)
-	}
-	if clusterAppPreReleaseString {
-		return preReleaseBuildFunc(), nil
+	if b.clusterApp != nil {
```

Comment:
This is unrelated fix, as the nil pointer panic got uncovered here as I updated the unit tests.